src/Repository.js

```diff
@@ -19,11 +20,30 @@
 
   LifeGamification.repository.updateSkills = function (skills) {
     return new Promise((resolve, reject) => {
-      chrome.storage.sync.clear(function() {
-        chrome.storage.sync.set({skillsCollectionId: skills}, function() {
-          resolve(skills);
-        });
+      chrome.storage.sync.set({skillsCollectionId: skills}, function() {
```

Comment:
In this change the whole application lose functionality of removing skills. We need tests to checks things like that.

Reply:
It doesn't. Those removed skills are overwritten.